test/fuzz/luaL_loadbuffer/serializer.cc

```diff
@@ -184,16 +184,98 @@ GetCondition(const std::string &counter_name, const std::string &then_block)
 	return retval;
 }
```

Comment:
commit message: 
please add a wording about introduced context and say that this commit is needed for two next commits.

Reply:
Fixed.

Reply:
Thanks!